modules/mobile/app/controllers/mobile/v0/check_in_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'chip/service'
+
+module Mobile
+  module V0
+    class CheckInController < ApplicationController
+      def create
+        response = chip_service.post_patient_check_in(appointment_ien: params[:appointmentIEN], patient_dfn:,
+                                                      station_no: params[:locationId])
+        attributes = JSON.parse(response.body)&.dig('data', 'attributes')
+        if (error = parse_error(attributes))
+          raise(*error)
+        else
+          render json: Mobile::V0::CheckInSerializer.new(@current_user.id, attributes)
+        end
+      end
+
+      def update
```

Comment:
for the demographics update, we have an existing demographics controller separate from the checkin controller that we should move this to.  `modules/mobile/app/controllers/mobile/v0/check_in_demographics_controller.rb`

Reply:
@jperk51 @aherzberg  Just for confirmation, I thought all check-in endpoints were going to be in one controller? Or is it better having it separated?

Reply:
I don't really have a strong preference. Whatever you all decide will be fine.

Reply:
If that check in demo controller is already in master we should just stick with that

Reply:
ok ill move it

Reply:
It'd be workable if demographics only had one endpoint but we'll have a show and update so we can't just use demographics custom action. it'd have to be show-demographics and update-demographics or something like that. and that'll break our convention.